homeassistant/components/mystrom/light.py

```diff
@@ -42,6 +47,23 @@
 )
 
 
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the myStrom entities."""
+    info = hass.data[DOMAIN][entry.entry_id]["info"]
+    device_type = info["type"]
+
+    if device_type == 102:
+        device = hass.data[DOMAIN][entry.entry_id]["device"]
+        try:
+            await device.get_state()
+        except MyStromConnectionError as err:
+            _LOGGER.error("No route to myStrom bulb: %s", info["ip"])
+            raise ConfigEntryNotReady() from err
+        async_add_entities([MyStromLight(device, info["mac"])])
+
+
 async def async_setup_platform(
```

Comment:
`async_setup_platform` should not create entities.
It should instead trigger an import and raise a `deprecated_yaml` issue in the issue registry.

Reply:
Implemented